package.nls.json

```
@@ -29,6 +29,7 @@
   "commands.SAS.upload": "Upload",
   "commands.SAS.uploadFiles": "Upload Files",
   "commands.SAS.uploadFolders": "Upload Folders",
+  "commands.SAS.toggleLineComment": "Toggle SAS code comments by line",
```

Oh sorry, I just noticed that the commands in VS Code are all camel case. We should follow the convention.
```suggestion
  "commands.SAS.toggleLineComment": "Toggle SAS Code Comments by Line",
```